avalon/tools/loader/widgets.py

```diff
@@ -522,7 +522,7 @@ def sorter(value):
                 error_info.append((
                     "Incompatible Loader",
                     None,
-                    representation["name"],
+                    representation.get("name"),
```

Comment:
This is already fixed in `2.x/develop` (and not like this).

Reply:
thanks. fixed